src/providers/on-chain-quote-provider.ts

```diff
@@ -663,6 +663,12 @@ export class OnChainQuoteProvider implements IOnChainQuoteProvider {
         _providerConfig?.blockNumber ?? originalBlockNumber + baseBlockOffset,
     };
 
+    log.info(`routes length: ${routes.length}`);
```

Comment:
are we sure we want to add these and the ones underneath?

Reply:
good catch - will remove - this was for local debugging logs